sdk/formrecognizer/azure-ai-formrecognizer/tests/test_business_card.py

```diff
@@ -418,6 +364,40 @@ def test_business_card_jpg_include_field_elements(self, client):
         for name, field in business_card.fields.items():
             for f in field.value:
                 self.assertFieldElementsHasValues(f.value_data.field_elements, business_card.page_range.first_page_number)
+        
```

Comment:
@kristapratico let me know what you're opinion on this is, I see pros and cons to combining these jpg tests into one. 
I will say that at the end of the day the test_business_card_jpg_include_field_elements already tests that a jpg can be passed to the endpoint successfully. However, I also think that the checks on the returned model are important which is why I've combined those here.